doc/sphinx/Makefile

```diff
@@ -60,22 +75,27 @@ woke-install:
 
 pa11y-install:
 	@type $(PA11Y) >/dev/null 2>&1 || { \
-			echo "Installing \"pa11y\" from npm... \n"; \
+			echo "Installing \"pa11y-ci\" from npm... \n"; \
 			mkdir -p $(SPHINXDIR)/node_modules/ ; \
-			npm install --prefix $(SPHINXDIR) pa11y; \
+			npm install --prefix $(SPHINXDIR) pa11y-ci; \
 		}
 
 install: $(VENVDIR)
+	sudo apt install -y $(DEPS)
 
-run: install
-	. $(VENV); sphinx-autobuild -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS)
+run: install configure
+	. $(VENV); cd $(WORKSPHINX); sphinx-autobuild -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS)
 
 # Doesn't depend on $(BUILDDIR) to rebuild properly at every run.
-html: install
-	. $(VENV); $(SPHINXBUILD) -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" -w $(SPHINXDIR)/warnings.txt $(SPHINXOPTS)
+html: install configure
+	# Don't rebuild html if already built
```

Comment:
Hmm… shouldn't this be a Make dependency, then?

Similar to `configure`, if it was a proper target that depended on another, we wouldn't configure again?

Reply:
Check the latest commit, does it match what you had in mind?

Reply:
Yeah, and then `configure` can be `$(WORKSPHINX)/CMakeCache.txt`

Reply:
It won't offer much speed improvements, just a few less lines of output

Reply:
Yeah, it's just The Way™ in Makefiles ;)

OTOH, unless you enumerate the sources/dependencies, I suppose that might just bit you in the foot since it won't update :)